### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by e-schultz**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (SethDavenport) [inline .vscode/settings.json]: does this belong in source control?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Angular/TypeScript/Webpack Starter
+
+This is the initial version of our starter project using Angular 1.x, TypeScript and Webpack to tie it all together.
```
Comment (SethDavenport) [inline README.md]: Angular 2 :)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* `npm start`: starts a production server serving the *dist* directory on [http://localhost:3000](http://localhost:3000)
+
+* `npm run build`: bundle all of the application including js/css/html files, with index.html generated according to a template specified in *index.html* (Everything will be put into *dist* folder).
+* `npm test`: will run the unit tests for the project as specified in *karma.conf.js* (everything ending in .test.ts will ge picked up, refer to *src/tests.entry.ts* if other extensions should be used).
+* `npm run e2e`: will run the e2e suite for this project located in *e2e* (refer to *wdio.conf.js* and *gulpfile.js* for more info, this is the only `gulp` dependency).
+* `npm run typings`: removes existing typings located in *typings* directory, reinstalls them based on *tsd.json*, and links whatever is available in *node_modules* (using `tsd link`).
+
+## Improvements
+
+This is an initial version of this setup and will be expanded in the future. Refer to the [issues section](https://github.com/rangle/ng-typescript-webpack-starter/issues) to see what needs to be done, or create a [new one](https://github.com/rangle/ng-typescript-webpack-starter/issues/new).
```
Comment (SethDavenport) [inline README.md]: Urls need updating?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## If something doesn't work
+
+Refer to the [issues section](https://github.com/rangle/ng-typescript-webpack-starter/issues) to see if this has already been logged. Otherwise create a [new issue](https://github.com/rangle/ng-typescript-webpack-starter/issues/new).
```
Comment (SethDavenport) [inline README.md]: Ditto

**Comment by SethDavenport** [comment]
Otherwise, :shipit:

File: .vscode/settings.json (hunk 1 above)
Comment (e-schultz) [inline .vscode/settings.json]: fixed - added to .gitignore and removed

**Pull request merged**